src/app/space/create/deployments/apps/deployments-apps.component.html

```diff
@@ -2,38 +2,60 @@
 </deployments-toolbar>
 
 <div *ngIf="(hasLoaded | async); else loading">
-  <div class="cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-xs-12 col-sm-4 col-md-2">
-        <h3>
-          <b>Applications</b>
-        </h3>
-      </div>
-      <div class="col-xs-12 col-sm-4 col-md-2">
-        <h3>
-          <b>Environments</b>
-        </h3>
+  <div *ngIf="applicationsList.length === 0">
```

Comment:
Could you use `*ngIf="applicationsList.length > 0; else emptyState" or something rather than two separate `ngIf`s with complementary conditions?